### PR TITLE
Rewrite EOS handle scene

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

- changed: Re-theme the EOS handle scene.

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205088953519810
  - https://app.asana.com/0/0/1205088126528660

### CODE REVIEW

**Comment by swansontec** [comment]
simulator_screenshot_60BB7A33-BB02-4631-9260-AFA9E3B32A45
simulator_screenshot_A9DCD6A7-62EE-4B67-9D38-A71ACF9B096F

**Comment by swansontec** [comment]
simulator_screenshot_8B8A596F-CA12-4317-9B43-22ED23BF9A09
simulator_screenshot_C1DA52EA-659B-4879-9110-9955320E2980

File: src/components/scenes/CreateWalletAccountSetupScene.tsx [hunk 1]
```diff
@@ -1,203 +1,104 @@
...
+      <MainButton
+        alignSelf="center"
+        disabled={spinning}
+        label={spinning ? undefined : lstrings.string_next_capitalized}
+        marginRem={0.5}
+        spinner={spinning}
+        type="primary"
+        onPress={handleSubmit}
+      />
+      <View style={{ paddingBottom: 400 }} />
```
Comment (Jon-edge) [inline src/components/scenes/CreateWalletAccountSetupScene.tsx:101]: This hard-coded padding allows you to scroll the scene for no reason, especially apparent when the keyboard is collapsed.

The MainButton ought to be stuck to the bottom of the screen or top of the keyboard.

**Review by Jon-edge — CHANGES_REQUESTED** [review]

File: src/components/scenes/CreateWalletAccountSetupScene.tsx [hunk 2]
```diff
@@ -1,203 +1,104 @@
...
+        autoCapitalize="none"
+        autoFocus
+        error={errorMessage}
+        label={lstrings.create_wallet_account_handle}
+        marginRem={[1, 1, 2]}
+        returnKeyType="next"
+        value={text}
+        onChangeText={handleChangeText}
+        onSubmitEditing={handleSubmit}
+      />
```
Comment (Jon-edge) [inline src/components/scenes/CreateWalletAccountSetupScene.tsx:91]: Add `maxLength={12}` counter

File: src/components/scenes/CreateWalletAccountSetupScene.tsx [hunk 3]
```diff
@@ -1,203 +1,104 @@
...
+      <MainButton
+        alignSelf="center"
+        disabled={spinning}
```
Comment (Jon-edge) [inline src/components/scenes/CreateWalletAccountSetupScene.tsx:94]: No reason to let the user continue if it didn't at least satisfy the 12 char requirement

**Review by Jon-edge — CHANGES_REQUESTED** [review]

File: src/components/scenes/CreateWalletAccountSetupScene.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/scenes/CreateWalletAccountSetupScene.tsx:101]: However, this isn't possible unless we 
1. Disable scroll in the SceneWrapper or
2. Implement content size aware (dynamic scroll enable/disable) SceneWrapper or
3. Some other trick I never figured out

I'm fine punting this detail until we do 2

**Review by Jon-edge — APPROVED** [review]

**Pull request merged**